### ISSUE TITLE
Replace Yomitan inline styles with CSS file

### ISSUE THREAD

**Issue by stephenmk** [stephenmk/Jitendex#104]
Yomitan recently added support for external CSS files, so inline styles are no longer necessary. This might greatly reduce the storage space consumed by Jitendex.

See: https://github.com/themoeway/yomitan/pull/1080

**Comment by stephenmk** [comment]
At least one roadblock: https://github.com/yomidevs/yomitan/issues/1500

**Comment by Lanwara** [comment]
Hey just wondering, if the inline styles disappear will Anki exports still have the required styles? Because I think it's great that this just works out of the box.

![anki_zubS55ZobX](https://github.com/user-attachments/assets/035596a9-7879-47c1-9b78-697419349686)

**Comment by stephenmk** [comment]
> if the inline styles disappear will Anki exports still have the required styles?

Correct, Anki exports will continue to work correctly by default. If a Yomitan dictionary stores CSS styles in a separate file rather than writing them inline, then Yomitan will embed the entire contents of the CSS file in each Anki flashcard it creates. (These CSS files usually aren't larger than 10KB in size, so the extra storage space required probably isn't a concern).

**Comment by stephenmk** [comment]
Bugs are still being discovered and resolved with Yomitan's CSS support. See [Yomitan issue 1768](https://github.com/yomidevs/yomitan/issues/1768)

**Comment by stephenmk** [comment]
Implemented in version 2025.08.01.0
